packages/web-api/src/methods.ts

```diff
@@ -471,30 +524,97 @@ export interface AdminAppsRestrictedListArguments extends WebAPICallOptions, Tok
   enterprise_id?: string;
 }
 cursorPaginationEnabledMethods.add('admin.apps.restricted.list');
-
+export interface AdminConversationsArchiveArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+}
+export interface AdminConversationsConvertToPrivateArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+}
+export interface AdminConversationsCreateArguments extends WebAPICallOptions, TokenOverridable {
+  is_private: boolean;
+  name: string;
+  description?: string;
+  org_wide?: boolean;
+  team_id?: string;
+}
+export interface AdminConversationsDeleteArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+}
+export interface AdminConversationsDisconnectSharedArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+  leaving_team_ids?: string | string[];
+}
+export interface AdminConversationsEKMListOriginalConnectedChannelInfoArguments
+  extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  channel_ids?: string | string[];
+  team_ids?: string | string[];
+}
+export interface AdminConversationsGetConversationPrefsArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+}
+export interface AdminConversationsGetTeamsArguments
+  extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  channel_id: string;
+}
+export interface AdminConversationsInviteArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+  user_ids: string | string[];
```

Comment:
Some of these I'm not sure based on the description if it takes either a `string` or a `string[]` or just one of those. @seratch is there a place in webapp that I can confirm these type of properties?

Reply:
@stevengill Even when passing a `string[]` value here, the value will be sent as a single string formatted by `JSON.stringify` in a form request body. https://github.com/slackapi/node-slack-sdk/blob/%40slack/web-api%405.12.0/packages/web-api/src/WebClient.ts#L385-L388

In other SDKs, we always generate a comma-separated format string value (not JSON string) from a list/array data. As far as I know, that's the safest way as the format is the only one mentioned in the API documentation. https://api.slack.com/methods/admin.conversations.invite

That said, doing `JSON.stringify` should always work as well. We've never encountered any issues with the current approach of this SDK.